server/auth/types/jwt/jwt_auth.test.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ *   Copyright OpenSearch Contributors
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import { getAuthenticationHandler } from '../../auth_handler_factory';
+
+describe('test jwt auth library', () => {
```

Comment:
Nit: I like smaller test functions that make it clearer what is being tested, could you refactor the call for `getAuthenticationHandler` to be a private function to make the test case read more like `Setup / Action / Verification`

Reply:
I created a helper function in this fixture called `getTestJWTAuthenticationHandlerWithConfig` to reduce some duplication and make the tests tighter.